.circleci/config.yml

```diff
@@ -136,44 +142,54 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - brew-{{ arch }}
+            - brew-macos-
           paths:
             - Library/Caches/Homebrew
       - run:
           name: install dev dependencies
           command: brew install cmake boost postgres grpc autoconf automake libtool ccache
       - save_cache:
-          key: brew-{{ arch }}-{{ epoch }}
+          key: brew-macos-{{ epoch }}
           paths:
             - Library/Caches/Homebrew
       - restore_cache:
           keys:
-            - build-cache-{{ arch }}-{{ .Branch }}
+            - build-cache-macos-{{ .Branch }}-
+            - build-cache-macos-
           paths:
             - ~/.ccache
       - restore_cache:
           keys:
-            - external-{{ arch }}-{{ .Branch }}
-            - external-{{ arch }}
+            - external-macos-{{ .Branch }}-
+            - external-macos-
           paths:
             - external
-      - run: ccache --show-stats
+      - run:
+          name: ccache setup
+          command: |
+            ccache --version
+            ccache --show-stats
```

Comment:
What's the point of showing stats in setup?

Reply:
For debug purposes.